pkg/ovs/openflow/ofctrl_group_test.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openflow
+
+import (
+	"net"
+	"testing"
+
+	"antrea.io/libOpenflow/openflow15"
+	"antrea.io/libOpenflow/util"
+	"antrea.io/ofnet/ofctrl"
+	"github.com/stretchr/testify/assert"
+)
+
+func getGroupMod(t *testing.T, g Group) *openflow15.GroupMod {
+	msgs, err := g.GetBundleMessages(AddMessage)
+	assert.NoError(t, err)
+	assert.Equal(t, 1, len(msgs))
+	return msgs[0].GetMessage().(*openflow15.GroupMod)
+}
+
+func TestBucketBuilder(t *testing.T) {
+	g := &ofGroup{ofctrl: &ofctrl.Group{}}
```

Comment:
Maybe move `g` into every case, then we could create a new ofGroup object, and no need to call `g.ResetBuckets()`

Reply:
Done